astropy/wcs/utils.py

```diff
@@ -1,22 +1,25 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
+import copy
+
 import numpy as np
 
-from astropy import units as u
+import astropy.units as u
+from astropy.coordinates import CartesianRepresentation, SphericalRepresentation, ITRS
 from astropy.utils import unbroadcast
-import copy
 
-from .wcs import WCS, WCSSUB_LONGITUDE, WCSSUB_LATITUDE
+from .wcs import WCS, WCSSUB_LATITUDE, WCSSUB_LONGITUDE
```

Comment:
My personal curiosity: why this change?

Reply:
I ran isort so it orders them alphabetically, I can undo it if you would like me to. I know we try to keep formatting changes out of PRs.

Reply:
No need, no need. I was just curious. See, I learned about `isort` :)